.github/workflows/rust-wasmedge-macro-release.yml

```diff
@@ -5,62 +5,83 @@ concurrency:
   cancel-in-progress: true
 
 on:
+  # workflow_dispatch:
+  #   branches:
+  #     - master
   push:
     branches:
-      - "master"
-      - "rust/release-wasmedge-macro"
-    paths:
-      - ".github/workflows/rust-wasmedge-macro-release.yml"
-      - "bindings/rust/wasmedge-macro/**"
-    tags:
-      - "rust-macro/[0-9]+.[0-9]+.[0-9]+*"
-  pull_request:
-    branches:
-      - master
-    paths:
-      - ".github/workflows/rust-wasmedge-macro-release.yml"
-      - "bindings/rust/wasmedge-macro/**"
+      - "ci/**"
```

Comment:
You should not only monitor the `ci/**` branches. There is anyone can submit a PR and modify this workflow file.